projects/cxfreeze.cmake

```diff
@@ -1,10 +1,15 @@
-ExternalProject_Add(cx_Freeze
-    #HG_REPOSITORY https://bitbucket.org/anthony_tuininga/cx_freeze
-    URL https://bitbucket.org/anthony_tuininga/cx_freeze/get/tip.tar.gz
-    CONFIGURE_COMMAND ""
-    BUILD_COMMAND ${PYTHON_EXECUTABLE} setup.py build
-    INSTALL_COMMAND ${PYTHON_EXECUTABLE} setup.py install --single-version-externally-managed --record=cxfreeze-install.log
-    BUILD_IN_SOURCE 1
-)
-
-SetProjectDependencies(TARGET cx_Freeze DEPENDS Python)
+# Using cx_Freeze installed via pip on Windows
+# Eg. "pip3 install cx_Freeze" or via .whl
+# Other OSs will build it manually...
+if(NOT BUILD_OS_WINDOWS)
```

Comment:
Apparently no possible since there are many problems with the combination of CMake+Windows.
Getting it working is postponed until we've got a working python-cmake-buildsystem.
But we already discussed that on IRC 😉